ipyleaflet/leaflet.py

```diff
@@ -536,13 +536,34 @@ class GeoJSON(FeatureGroup):
     style = Dict().tag(sync=True)
     hover_style = Dict().tag(sync=True)
     point_style = Dict().tag(sync=True)
+    style_callback = Any()
 
     _click_callbacks = Instance(CallbackDispatcher, ())
     _hover_callbacks = Instance(CallbackDispatcher, ())
 
+    @observe('data', 'style', 'style_callback')
+    def _update_data(self, change):
```

Comment:
It seems to shadow the `Choropleth._update_data` method. Maybe rename `Choropleth._update_data` to `Choropleth._update_choropleth_data` ?

Reply:
Update name to _update_choropleth_data